src/Servers/test/FunctionalTests/HelloWorldTest.cs

```diff
@@ -108,4 +109,68 @@ public async Task HelloWorld(TestVariant variant)
             }
         }
     }
+
+    public static TestMatrix SelfHostTestVariants
+        => TestMatrix.ForServers(ServerType.Kestrel, ServerType.HttpSys)
+            .WithTfms(Tfm.Default)
+            .WithApplicationTypes(ApplicationType.Portable)
+            .WithAllHostingModels()
+            .WithAllArchitectures();
+
+    [ConditionalTheory]
+    [MemberData(nameof(SelfHostTestVariants))]
+    public async Task ApplicationException(TestVariant variant)
+    {
+        var testName = $"ApplicationException_{variant.Server}_{variant.Tfm}_{variant.Architecture}_{variant.ApplicationType}";
+        using (StartLog(out var loggerFactory,
+            variant.Server == ServerType.Nginx ? LogLevel.Trace : LogLevel.Debug, // https://github.com/aspnet/ServerTests/issues/144
+            testName))
```

Comment:
Nginx isn't used for this test.
```suggestion
        using (StartLog(out var loggerFactory, LogLevel.Debug, testName))
```